ballerina-tests/tests/47_unary_timestamp_service.bal

```diff
@@ -22,39 +22,39 @@ listener grpc:Listener ep47 = new (9147);
 @grpc:ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_47_UNARY_TIMESTAMP, descMap: getDescriptorMap47UnaryTimestamp()}
 service "TimestampService" on ep47 {
 
-    remote function getGreeting(TimestampServiceGreetingCaller caller, string value) returns error? {
+    remote function getGreeting(TimestampServiceGreetingCaller caller, string value) {
         Greeting greeting = {
             name: value,
-            time: check time:utcFromString("2007-12-03T10:15:30.120Z")
+            time: checkpanic time:utcFromString("2007-12-03T10:15:30.120Z")
```

Comment:
```suggestion
            time: check time:utcFromString("2007-12-03T10:15:30.120Z")
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/5db9dbc40838de6a15d5c28f2d02986dcb8079fc